guides/doc-Provisioning_Guide/topics/Virt-RHV.adoc

```diff
@@ -85,14 +107,14 @@ The items listed in this step are provided as Technology Previews. For further i
 
 .For CLI Users
 
-To create a Red Hat Virtualization connection, enter the `hammer compute-resource create` command with `Ovirt` for `--provider` and the name of the data center you want to use for `--datacenter`.
+To create a {OVirt} connection, enter the `hammer compute-resource create` command with `Ovirt` for `--provider` and the name of the data center you want to use for `--datacenter`.
 
 [options="nowrap" subs="+quotes"]
 ----
 # hammer compute-resource create \
---name "_My_RHV_" --provider "Ovirt" \
---description "RHV server at _rhvm.example.com_" \
---url "_https://rhvm.example.com/ovirt-engine/api_" \
+--name "_My_OVirt_" --provider "Ovirt" \
+--description "OVirt server at _ovirt.example.com_" \
```

Comment:
Could we not just use the variable for {Ovirt} here @lzap would that not just work, rather than making changes?

Reply:
That's a good idea.

Reply:
@lzap, all is done except the ovirt.example.com's through the file. If you're too busy, I can do it.

Reply:
You say "you're busy" one more time and I quit! ;-) Lemme fix that.